src/copyrightinserter.ts

```diff
@@ -21,10 +21,28 @@ export class CopyrightInserter {
     // map of supported license labels to inline formatting functions that
     // insert copyright holder and year into the license
     // currently supported licenses are:
+    // 'agl3' - AGPL 3.0
     // 'apache' - APACHE 2.0
     // 'bsd' - BSD
+    // 'gpl3' - GPL 3.0
     // 'mit' - MIT
     readonly CopyrightMap = new Map([
+        ['agpl3', (holder: string, year:string) =>
+`Copyright (C) ${year} ${holder}
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.`],
+
```

Comment:
Please, add the new licenses at the end. There is no need to sort the map initialization data.
The current change creates a cognitive challenge to a reviewer and makes it harder to follow up the change.
Having new map elements added to the end simplifies the diff of the reviewed commit history.